### ISSUE TITLE
GetMatches and rate limiting

### ISSUE THREAD

**Issue by SeaRoth** [kevinohashi/php-riot-api#71]
Is there a timeline for completion of the $api->getMatches($ids, $timeline=true) ?

**Comment by dakotawashok** [comment]
@SeaRoth  It should already be done, I'm getting the timeline in my match calls, are you not?

**Comment by SeaRoth** [comment]
Yeah, maybe I'm doing something wrong, both getMatches and getMatch are returning null.

## Output:
![image](https://user-images.githubusercontent.com/3461713/33904743-5a8b70e6-df31-11e7-91bd-d498b2c3b5ed.png)

## Code:

```
$api = new riotapi('NA1', new FileSystemCache('cache/'));
$name = "The SeaRoth";
$id = 32493361;
$theMatchList;

try {
    $theMatchList = $api->getMatchList($id,$params=null);
} catch(Exception $e) {
    echo "Error: " . $e->getMessage();
};

$match1 = $theMatchList['matches'][0]['gameId'];
$match2 = $theMatchList['matches'][1]['gameId'];
echo "Match #1: " . $match1 . "<br>";
echo "Match #2: " . $match2 . "<br>";
echo "<br><br>";
echo "matches";
echo "<br><br>";

try{
    $rMatches = $api->getMatches([$match1,$match2], $includeTimeline = true);
    print_r(json_encode($rMatches));
}catch(Exception $e){
    echo "Error: " . $e->getMessage();
}

echo "<br><br>";
echo "match";
echo "<br><br>";
try{
    $rMatch = $api->getMatch($match1,false);
    print_r(json_encode($rMatch));
}catch(Exception $e){
    echo "Error: " . $e->getMessage();
}
```

**Comment by dakotawashok** [comment]
I'll take a look at it later tonight after work, but that looks like it should be working

**Comment by SeaRoth** [comment]
Thanks boss! xD

**Comment by dakotawashok** [comment]
@SeaRoth Looks like you're right! getMatches isn't working, and it looks like it's because of a currently undefined method on line 183 of php-riot-api.php called requestMultiple(). I've altered your code to work around this: 

`$name = "The SeaRoth";
        $id = 32493361;
        $theMatchList;

        try {
            $theMatchList = $this->api->getMatchList($id,$params=null);
        } catch(Exception $e) {
            $this->log("Error: " . $e->getMessage());
        };

        $match1 = $theMatchList->matches[0]->gameId;
        $match2 = $theMatchList->matches[1]->gameId;
        $this->log("Match #1: " . $match1);
        $this->log("Match #2: " . $match2);
        $this->log("matches");

        try{
            $definedMatches = [];
            foreach([$match1, $match2] as $gameId) {
                $rMatch = $this->api->getMatch($gameId,false);
                $this->log($rMatch);
                array_push($definedMatches, $rMatch);
            }
        }catch(Exception $e){
            $this->log("Error: " . $e->getMessage());
        }`

and the output works. Note that I've replaced your echos with my log function, but they both do the same thing :)
